backend/game/views/player.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import viewsets, status, mixins
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from django.shortcuts import get_object_or_404
+from game.models import Game, Player
+from game.serializers import PlayerSerializer
+from users.models import User
+
+
+class PlayerViewSet(mixins.CreateModelMixin,
+                    mixins.RetrieveModelMixin,
+                    mixins.DestroyModelMixin,
+                    mixins.ListModelMixin,
+                    viewsets.GenericViewSet):
+    queryset = Player.objects.all()
+    serializer_class = PlayerSerializer
+    permission_classes = [IsAuthenticated]
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        # Retrieve game and user, return a 404 response if at least one doesn't exist
+        game_id, user_id = serializer.data.get('game'), serializer.data.get('user')
+        game, user = get_object_or_404(Game, pk=game_id), get_object_or_404(User, pk=user_id)
+
+        # Check if user doesn't already controls a player in game, and if user is the author of the request
+        if game.players.filter(user__username=user.username) or request.user.id != user_id:
```

Comment:
Est-ce qu'in a vraiment besoin de vérifier ça nous-mêmes ? Qu'est-ce que DRF renvoie si on fournit un identifiant de Game invalide ?